helm/csi-powerstore/Chart.yaml

```diff
@@ -2,7 +2,7 @@ name: csi-powerstore
 apiVersion: v2
 version: 2.3.0
 appVersion: "2.3.0"
-kubeVersion: ">= 1.21.0 <= 1.24.0"
+kubeVersion: ">= 1.21.0 <= 1.25.0"
```

Comment:
@harishp8889 Shouldn't it be <1.25 if we adding support upto 1.24 ? Or < 1.26 if we are adding support for 1.25 as well?

Reply:
done